kyuubi-server/src/main/scala/org/apache/kyuubi/engine/ApplicationOperation.scala

```diff
@@ -143,4 +153,21 @@ object ApplicationManagerInfo {
       resourceManager,
       KubernetesInfo(kubernetesContext, kubernetesNamespace))
   }
+
+  def serialize(appMgrInfo: ApplicationManagerInfo): String = {
+    Base64.getEncoder.encodeToString(
+      mapper.writeValueAsString(appMgrInfo).getBytes(StandardCharsets.UTF_8))
+  }
+
+  def deserialize(encodedStr: String): ApplicationManagerInfo = {
+    try {
+      info(s"The original string encoded:$encodedStr")
```

Comment:
move to catch block. we only need to print the base64 string when something goes wrong

Reply:
Thanks. Done